export_schema/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.26.0"
 authors = ["Dave Kozma <dkozma@adobe.com>"]
 license = "MIT OR Apache-2.0"
 edition = "2018"
-rust-version = "1.65.0"
+rust-version = "1.70.0"
```

Comment:
FWIW when we're doing an MSRV change, we should be changing this value. Looks like the MSRV was changed in documentation and CI while I was away, but we missed this value.